backend/src/main/java/com/bakdata/conquery/models/jobs/ImportJob.java

```diff
@@ -113,10 +88,20 @@ public static ImportJob create(
 			header.assertMatch(table);
 
 			final ImportId importId = new ImportId(table.getId(), header.getName());
+			Import processedImport = namespace.getStorage().getImport(importId);
 
-			if (namespace.getStorage().getImport(importId) != null) {
+			if (update) {
+				if (processedImport == null) {
```

Comment:
Finde ich ehrlich gesagt unproblematisch, wenn der noch nicht da ist. @thoniTUB wie siehst du das?

Reply:
ich finde schon wichtig, denn derjenige,  der die Update-Request schickt, muss klar sein, was das bedeutet.

Reply:
Ich möchte es auch so lassen , weil ich das ganze mehr RESTful gestallten möchte, dafür müssen die Imports ihre Abhängigkeit von Table und Tag verlieren ;)

Reply:
Ah mist, ich sehe das jetzt erst, der Endpunkt ist ja auf Ebene der Table. Dann muss ich ja beim reinladen immer noch die Table wissen dadurch ist in der API eigentlich nichts gewonnen :/ Sorry hatte ich davor leider überlesen